packages/catlog/src/tt/mod.rs

```diff
@@ -352,11 +352,14 @@
 //! be semantically the same type.
 
 pub mod batch;
-pub mod elab;
+pub mod context;
 pub mod eval;
 pub mod modelgen;
+#[allow(unused)]
+pub mod notebook_elab;
```

Comment:
Can you remove `allow(unused)` here?

Reply:
Yes, I believe so, I just had that on in development so it wouldn't be annoying.